Formula/osmcoastline.rb

```diff
@@ -0,0 +1,33 @@
+class Osmcoastline < Formula
+  desc "Extracts coastline data from OpenStreetMap planet file"
+  homepage "https://osmcode.org/osmcoastline/"
+  url "https://github.com/osmcode/osmcoastline/archive/v2.3.0.tar.gz"
+  sha256 "b08ddcc8cb494a19cbf4d9f1501c47443fe374a4fe171e6f55c730ca1e710689"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```
Seems like this is `-or-later`. See https://github.com/osmcode/osmcoastline/search?q=license

Reply:
Thank you for you suggestion.
+    license "GPL-3.0-or-later"